.browserslistrc

```diff
@@ -0,0 +1 @@
+last 1 version, >2%
```

Comment:
Is there a reason why we are dropping firefox ESR ?

Reply:
I've taken the list of supported browsers from the documentation: https://support.bigcommerce.com/s/article/Themes-Supported-Browsers. Firefox ESR stands for _firefox 52_
<img width="626" alt="Screenshot 2020-09-14 at 14 28 13" src="https://user-images.githubusercontent.com/68893868/93080624-ad39e000-f696-11ea-8311-29a49f0b62ed.png">

Reply:
Changing this to `>2%` is a big change. We should check with @bookernath @bc-as before we make this change. Also we should stick with the last 2 versions.

Reply:
last 2 versions > will mean, that we will include IE 10 support and polyfills for it. The doc https://support.bigcommerce.com/s/article/Themes-Supported-Browsers also says, that we are targeting more then 2% browsers

Reply:
Agreed and I just want a confirmation from product that the doc is right or if it needs updating

Reply:
Lets go with: >2%, last 1 version, Firefox ESR